instrumentation/logger/Appraisals

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# # Copyright The OpenTelemetry Authors
+# #
+# # SPDX-License-Identifier: Apache-2.0
+
+# # TOOD: Re-enable before release, along with active_support_logger tests
+# appraise 'rails-6.0' do
+#   gem 'rails', '~> 6.0.0'
+# end
+
+# appraise 'rails-6.1' do
+#   gem 'rails', '~> 6.1.0'
+# end
+
+# appraise 'rails-7.0' do
+#   gem 'rails', '~> 7.0.0'
+# end
```

Comment:
I haven't been able to get Appraisal to work with gems installed from git source. This might just be due to my limited knowledge of Appraisal. We may need to wait until the opentelemetry-logs-related gems are released before we can re-instate these.

Reply:
Have you tried something like:
```
gem 'opentelemetry-logs-api', github: 'open-telemetry/opentelemetry-ruby', branch: 'main', glob: 'logs_api/opentelemetry-logs-api.gemspec'
gem 'opentelemetry-logs-sdk', github: 'open-telemetry/opentelemetry-ruby', branch: 'main', glob: 'logs_sdk/opentelemetry-logs-sdk.gemspec'
```